laravel/app/Http/Controllers/ScheduleDataController.php

```diff
@@ -581,270 +433,89 @@ private function insertOption()
-                        "attending" => $attending,
-                        "milestones" => $_REQUEST["milestones2"],
-                        "objectives" => $_REQUEST["objectives2"],
-                        "anesthesiologist_id" => $pref_anest2,
-                        "isValid" => 1,
-                    ]);
-            } else {
-                Option::where("date", $date)
-                    ->where("resident", $resident)
-                    ->where("option", $choice)
-                    ->delete();
-            }
-        } else {
-            $overwrittenChoices[1] = 0;
-            if (!is_null($schedule_data2)) {
-                // Insert data
-                Option::insert([
-                    "date" => $date,
-                    "resident" => $resident,
-                    "schedule" => $split[1],
-                    "attending" => $attending,
-                    "option" => $choice,
-                    "milestones" => $_REQUEST["milestones2"],
-                    "objectives" => $_REQUEST["objectives2"],
-                    "anesthesiologist_id" => $pref_anest2,
-                    "isValid" => 1,
-                ]);
-            }
-        }
-
-        //Third choice data
-        $choice++;
-        $schedule_data3 = null;
-        $attending = null;
-        // Insert third choice data if it exists
-        if ($split[2] != 0) {
-            $schedule_data3 = ScheduleData::where("id", $split[2])->get();
-            // Get attending id
-            $attending_string = $schedule_data3[0]["lead_surgeon"];
-            $attending = substr(
-                $attending_string,
-                strpos($attending_string, "[") + 1,
-                strpos($attending_string, "]") - (strpos($attending_string, "[") + 1)
-            );
-        }
-
-        if (isset($_REQUEST["pref_anest3"])) {
-            if ($_REQUEST["pref_anest3"] != 0) {
-                $pref_anest3 = $_REQUEST["pref_anest3"];
-            }
-        }
+        for ($i = 0; $i < 3; $i++) {
+            $choice = $i + 1;
+            if ($split[$i]) {
+                $pref_anest[$i] = null;
+                $schedule_data[$i] = ScheduleData::where("id", $split[$i])->get();
+                $attending_string = $schedule_data[$i][0]["lead_surgeon"];
+                $date = $schedule_data[$i][0]["date"];
+                $attending = substr(
+                    // get rid of [number] after the surgeon name
+                    $attending_string,
+                    strpos($attending_string, "[") + 1,
+                    strpos($attending_string, "]") - (strpos($attending_string, "[") + 1)
+                );
+
+                if (isset($_REQUEST["pref_anest" . $choice]) && $_REQUEST["pref_anest" . $choice] != 0) {
+                    // if they chose an anesthesiologist, add their ID to the DB, if not, add NULL
+                    $pref_anest[$i] = $_REQUEST["pref_anest" . $choice];
+                }
```

Comment:
I know this way is consistent w/ the rest of REMODEL, but I'd prefer we use Laravel's Request class, instead of `$_REQUEST`.

Thoughts?

Reply:
Could you send documentation for the method you're looking for?

Reply:
https://laravel.com/docs/8.x/requests#accessing-the-request

Reply:
As much as I'd like to get this change made, we've decided (on other forms of communication) to let it slide in the interest of time.